spec/stack.js

```diff
@@ -3,20 +3,55 @@ import chaiChange from 'chai-change'
 import Stack from '../src/stack'
 
 chai.use(chaiChange)
-
 describe('Stack', () => {
-  'use strict'
+    'use strict'
+
+    it('exists', () => {
+        expect(Stack).to.be.a('function')
+    })
+
+    context('push()', () => {
+        it('pushes an element to the top of the stack.', () => {
+            const myStack = new Stack()
+            expect(() => myStack.pushie('foo'))
```

Comment:
`pushie` is not the name of the function required by the spec.  `push` is *the* name for this behavior in a `Stack` data structure.